src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilTest.java

```diff
@@ -125,9 +125,9 @@ public void testTokenValueIncorrect() throws IllegalAccessException {
             TokenUtil.getTokenName(nextAfterMaxId);
             fail("IllegalArgumentException is expected");
         }
-        catch (IllegalArgumentException expected) {
```

Comment:
Misordered 'assertEquals()' arguments (false positive). Fixed by renaming the parameter.